src/cpp/core/system/Win32System.cpp

```diff
@@ -958,10 +958,69 @@ CloseHandleOnExitScope::~CloseHandleOnExitScope()
    }
 }
 
-Error terminateChildProcesses()
+struct ProcessInfo
+{
```

Comment:
Do we need a default constructor that initializes these to "null" values (i.e. 0 or -1). Fine if we don't, just asking the question.